docs/tasks/administer_cluster_quotas.md

```diff
@@ -210,6 +210,168 @@ To create the ClusterQueue, run the following command:
 kubectl apply -f cluster-total.yaml
 ```
 
+## Multiple ClusterQueues and borrowing cohorts
+
+Two or more ClusterQueues can borrow unused quota from other ClusterQueues in
+the same cohort.
+
+To establish a cohort, set a common name in the `.spec.cohort` field in all the
+ClusterQueues you want to include in the cohort.
```

Comment:
Proposal [218 - 222]:

With the following example, you can establish  and set a common name in the `.spec.cohort` field in all the
ClusterQueues you want to include in the cohort.

Reply:
Applied with a tweak